airflow/www/views.py

```diff
@@ -2288,6 +2292,53 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except :
+            pass
+
+        # Get the Git repo and git hash : first try pip freeze, then look at git
+        git_repo = git_hash = None
+        try:
+            p = subprocess.Popen('pip freeze | grep airflow',
```

Comment:
I'm a little bit uncomfortable about adding a dependency to PIP (also it's possible that the running version of airflow doesn't match this version if e.g. it has a different PYTHONPATH). One idea is to show the hard-coded airflow version plus have a version file generated by setup-tools if this is possible indicating the SHA of the repo used to generate the package if relevant (this can be in v2). Since Airflow is aware of where it is running from it would always use the right "version" file.